DirectProgramming/DPC++FPGA/include/utils.hpp

```diff
@@ -1,46 +1,7 @@
-#pragma once
+#ifndef __UTILS_HPP__
```

Comment:
I don't think this file belongs in the common include directory

Reply:
I've replaced this with the use of NTuples, which will be in the include directory